ros/resources/v0/hosts.py

```diff
@@ -90,6 +91,8 @@ def get(self):
                 host['id'] = i.__dict__['id']
                 host['recommendation_count'] = 5
                 host['state'] = 'Undersized'
+                pro = PerformanceProfile()
+                pro.my_score
```

Comment:
```suggestion
                host['my_score'] = i.my_score
```
With this, you will get value inside response.